pywbem/mof_compiler.py

```diff
@@ -726,6 +726,13 @@ def p_mp_createClass(p):
             p.parser.log(
                 _format("Error modifying class {0!A}: {1}, {2}",
                         cc.classname, ce.status_code, ce.status_description))
+            raise MOFRepositoryError(
+                msg=_format(
+                    "Cannot compile class {0!A} because the CIM repository "
```

Comment:
Since we are fixing, this msg bothers me slightly because the first user question is why is the error on ModifyClass. Maybe slight extension to message that compiled class previously exists in in repository.

Reply:
DONE - changed to:
"Cannot compile class {0!A} because the class already existed in the CIM repository and ModifyClass failed"